src/account/AccountStorage.sol

```diff
@@ -68,6 +70,8 @@ struct AccountStorage {
     mapping(bytes4 => uint256) supportedIfaces;
     // Installed plugins capable of signature validation.
     EnumerableSet.Bytes32Set signatureValidations;
+    // Todo: merge this with other validation storage?
+    EnumerableSet.Bytes32Set defaultValidations;
```

Comment:
TODO idea: there a way to merge 2 LLSets into 1? 

`SENTINEL_VALUE = 1`
`SENTINEL_VALUE_2 = 2`

`SENTINEL_VALUE` -> `LLSet1_elem_1` -> ... -> `LLSet1_elem_i` -> `SENTINEL_VALUE_2` -> `LLSet2_elem_1` -> ... `LLSet2_elem_i` -> `SENTINEL_VALUE`

Reply:
nvm - although this gives an o(1) existence check across 2 sets, you wouldn't be able to tell which LLSet the elem belongs in

Reply:
We want a set here because it's 1 `SLOAD` for UOs even if we do a single `FunctionReference` right? And for runtime we don't check this

Reply:
Yep, and I ended up refactoring it in #64 to use a shared `ValidationData` field in account storage.